pkg/provider/aws/action/openshift-snc/cloud-config

```diff
@@ -0,0 +1,39 @@
+#cloud-config
+runcmd:
+  - systemctl enable --now kubelet
+  - pushd /tmp
+  - curl "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m).zip" -o "awscliv2.zip"
+  - podman run -it --rm --workdir /data --privileged -v ${PWD}:/data registry.access.redhat.com/ubi9/toolbox unzip awscliv2.zip
+  - ./aws/install
+  - export PS=$(aws ssm get-parameter --name "{{ .SSMPullSecretName }}" --with-decryption --query "Parameter.Value" --output text)
+  - echo ${PS} > /opt/crc/pull-secret
+  - chmod 0644 /opt/crc/pull-secret
+  - export CA64=$(aws ssm get-parameter --name "{{ .SSMCaCertName }}" --with-decryption --query "Parameter.Value" --output text)
+  - echo ${CA64} > /opt/crc/custom-ca.crt
+  - chmod 0644 /opt/crc/custom-ca.crt
+  - export KP=$(aws ssm get-parameter --name "{{ .SSMKubeAdminPasswordName }}" --with-decryption --query "Parameter.Value" --output text)
+# Check this when new bundle created
+  - echo ${KP} > /opt/crc/password_kubeadmin
+  - chmod 0644 /opt/crc/password_kubeadmin
```

Comment:
note: `password_kubeadmin` file is not needed anymore, `pass_kubeadmin` is used by all the services that needs the password

Reply:
thx for the reminder, removed!